scripts/contractInteraction/staking_vesting.py

```diff
@@ -415,4 +438,16 @@ def addVestingCodeHash(vestingLogic):
     data = staking.addContractCodeHash.encode_input(vestingLogic)
     sendWithMultisig(conf.contracts['multisig'], staking.address, data, conf.acct)
 
-
+# addresses is vesting addresses that will be set
+# vestingCreationAndTypes is the array of VestingCreationAndTypeDetail struct
+# struct VestingCreationAndTypeDetail {
+#         bool isSet;
+#         uint120 vestingType;
+#         uint128 vestingCreationType;
+#     }
+# make sure the length of addresses & vestingCreationAndTypeDetails is the same
+def registerVestingToVestingCreationAndTypes(addresses, vestingCreationAndTypeDetails):
+    if len(addresses) != len(vestingCreationAndTypeDetails) print("umatched length of array between addresses & vestingCreationAndTypeDetails") return
```

Comment:
1) pls check syntax - the linter says missing column for `if` condition  
2) it is more conventional to use `raise Exception("umatched length of array between addresses & vestingCreationAndTypeDetails")`